docs/src/man/split_apply_combine.md

```diff
@@ -400,20 +511,121 @@ To also get the values of the grouping columns along with each group, use the
 `pairs` function:
 
 ```jldoctest sac
-julia> for (key, subdf) in pairs(groupby(iris, :Species))
+julia> for (key, subdf) in pairs(iris_gdf)
            println("Number of data points for $(key.Species): $(nrow(subdf))")
        end
 Number of data points for Iris-setosa: 50
 Number of data points for Iris-versicolor: 50
 Number of data points for Iris-virginica: 50
 ```
 
-The value of `key` in the previous example is a [`DataFrames.GroupKey`](@ref) object,
+Note that although `GroupedDataFrame` is iterable and indexable it is not an
+`AbstractVector`. For this reason currently it was decided that it does not
+support `map` nor broadcasting (to allow for making a decision in the future
+what result type they should produce). To apply a function to all groups of a
+data frame and get a vector of results either use a comprehension or `collect`
+`GroupedDataFrame` into a vector first. Here are examples of both approaches:
+
+```jldoctest sac
+julia> [nrow(sdf) for sdf in iris_gdf]
```

Comment:
> >    Still relevant.
>
>@nalimilan - what do you mean here? Could you please clarify?
>
>(GitHub does not show me what you refer to). You wanted me to put together all examples with printing >number of rows for all groups in one place and I did this.

Ah yes the GitHub UI is quite poor for that. It was https://github.com/JuliaData/DataFrames.jl/pull/3225/files/b4560f3c171048ff41107b7e78492c82a8be163b#r1027354724.

The example here would be better at the end of the block.

Reply:
@nalimilan - I have moved this example and added some more comments. I hope now it is better 😄.